File: 2977-check-if-a-string-is-an-acronym-of-words/2977-check-if-a-string-is-an-acronym-of-words.c
bool isAcronym(char** words, int wordsSize, char* s) {
  int n=wordsSize;
  char *s2=(char*)malloc((n+1)*sizeof(char));
  for(int i=0;i<n;i++)
  {
    s2[i]=words[i][0];
  }  
  s2[n]='\0';
  if(strcmp(s,s2)==0)
  return true;
  else
  return false;
}

File: 2977-check-if-a-string-is-an-acronym-of-words/2977-check-if-a-string-is-an-acronym-of-words.java
class Solution {
    public boolean isAcronym(List<String> words, String s) {
        StringBuilder m=new StringBuilder();
        int n=words.size();
        for(int i=0;i<n;i++)
        {
            m.append(words.get(i).charAt(0));
        }
        return s.equals(m.toString());
    }
}